lib/qs/payload_handler.rb

```diff
@@ -88,9 +88,9 @@ def log_complete(redis_item)
         'time'    => redis_item.time_taken,
         'handler' => redis_item.handler_class
       }
-      if (job = redis_item.job)
-        summary_line_args['job']    = job.name
-        summary_line_args['params'] = job.params
+      if (message = redis_item.message)
+        summary_line_args['name']   = message.route_id
+        summary_line_args['params'] = message.params
```

Comment:
@jcredding it is cool that we can just log params straight up and it works whether the message is a job or event.